data-prepper-plugins/saas-source-plugins/jira-source/src/main/java/org/opensearch/dataprepper/plugins/source/saas/jira/JiraSource.java

```diff
@@ -0,0 +1,57 @@
+package org.opensearch.dataprepper.plugins.source.saas.jira;
+
+
+import org.opensearch.dataprepper.metrics.PluginMetrics;
+import org.opensearch.dataprepper.model.acknowledgements.AcknowledgementSetManager;
+import org.opensearch.dataprepper.model.annotations.DataPrepperPlugin;
+import org.opensearch.dataprepper.model.annotations.DataPrepperPluginConstructor;
+import org.opensearch.dataprepper.model.buffer.Buffer;
+import org.opensearch.dataprepper.model.codec.ByteDecoder;
+import org.opensearch.dataprepper.model.event.Event;
+import org.opensearch.dataprepper.model.plugin.PluginFactory;
+import org.opensearch.dataprepper.model.record.Record;
+import org.opensearch.dataprepper.model.source.Source;
+import org.opensearch.dataprepper.plugins.source.saas_crawler.SaasCrawlerApplicationContextMarker;
+import org.opensearch.dataprepper.plugins.source.saas_crawler.base.Crawler;
+import org.opensearch.dataprepper.plugins.source.saas_crawler.base.SaasPluginExecutorServiceProvider;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * JiraConnector connector entry point.
+ */
+
+@DataPrepperPlugin(name = "jira",
+        pluginType = Source.class,
+        packagesToScan = {SaasCrawlerApplicationContextMarker.class, JiraSource.class}
+)
+public class JiraSource implements Source<Record<Event>> {
```

Comment:
Is this supposed to inherit from `SaasSourcePlugin`?

Reply:
As we discussed, yes it will. But I didn't do that to limit the size of this PR.